tests/cross/src/test/scala/tests/pc/CompletionDocSuite.scala

```diff
@@ -608,7 +654,7 @@ class CompletionDocSuite extends BaseCompletionSuite {
     includeDocs = true
   )
   check(
-    "scala11",
+    "scala11".tag(IgnoreScala3),
```

Comment:
What's the problem with Scala3?

Reply:
New completions are not yet implemented. I will add a comment about it.

Reply:
Added!